### PR TITLE
Add bios autoboot option

### PR DESCRIPTION
**Pull request by Cpasjuste**
This PR allow booting directly to the bios by giving the "bios" arg instead an image file.

### CODE REVIEW

File: libswirl/gui/gui.cpp [hunk 1]
```diff
@@ -354,6 +354,8 @@ struct ReicastUI_impl : GUI {
             std::string game_file = cfgLoadStr("config", "image", "");
             if (!game_file.empty())
             {
+                if(game_file == "bios")
```
Comment (skmp) [inline libswirl/gui/gui.cpp:357]: Do you mind adjusting the logic in `libswirl.cpp::int GetFile(char* szFileName)`? You'll have to change a few empty strings as well in GUI, but it's a much cleaner solution to make "bios" mean "bios" at the core level. I never liked using an empty string anyway.

**Review by skmp — CHANGES_REQUESTED** [review]

File: libswirl/gui/gui.cpp (hunk 1 above)
Comment (skmp) [reply libswirl/gui/gui.cpp:357]: Hmm, also maybe name it no "nodisk" to make it clear what it does?

**Review by baka0815 — CHANGES_REQUESTED** [review]
@Cpasjuste you should add a parameter ``-bios``/``--bios`` (or ``-nodisk``/``--nodisk``) like in line 127 of cl.cpp instead of using "bios" as a magic value for a disk image.

**Comment by skmp** [comment]
@baka0815 I don't mind using a special value for no disk, but I'd rather it be "nodisk" than "bios". Command line arguments aren't settable from the cfg file, that's why I try to keep them at a minimum

**Comment by skmp** [comment]
functionality merged via #1818

**Pull request closed without merging**